app/helpers/schools_helper.rb

```diff
@@ -0,0 +1,2 @@
+module SchoolsHelper
```

Comment:
This is not a real review... jsut practice...

I wonder why we need this empty class..

Reply:
We probably don't, I used `rails g` which spawned a whole smorgasbord of probably unnecessary files, but don't know quite enough to tell the difference.